circuit_knitting_toolbox/circuit_cutting/cutting_evaluation.py

```diff
@@ -14,9 +14,110 @@
 from __future__ import annotations
 
 from collections.abc import Sequence
+from itertools import chain
+from multiprocessing.pool import ThreadPool
 
+import numpy as np
 from qiskit.circuit import QuantumCircuit, ClassicalRegister
-from ..utils.observable_grouping import CommutingObservableGroup
+from qiskit.quantum_info import PauliList
+from qiskit.providers import Backend
+from qiskit_aer import AerSimulator
+
+from ..utils.observable_grouping import CommutingObservableGroup, ObservableCollection
+from ..utils.iteration import strict_zip
+from .qpd import (
+    QPDBasis,
+    SingleQubitQPDGate,
+    TwoQubitQPDGate,
+    generate_qpd_samples,
+    decompose_qpd_instructions,
+)
+from .cutting_decomposition import decompose_observables
+
+
+def execute_experiments(
```

Comment:
@garrison I couldn't get Sphinx to understand the indentions in the args list without adding ticks